src/components/forms/language-panel/language-panel.js

```diff
@@ -108,7 +108,7 @@ LanguagePanel.propTypes = {
         PropTypes.object
       ])
     ),
-    touched: PropTypes.objectOf(PropTypes.bool),
+    touched: PropTypes.oneOfType([PropTypes.bool, PropTypes.object]),
```

Comment:
I've already fixed this console error in my PR, so you can discard this change to prevent conflicts when merge